test/typescript/terraform-cloud/main.ts

```diff
@@ -26,6 +27,10 @@ export class HelloTerra extends TerraformStack {
         token
       }
     });
+
+    new TerraformAsset(this, "asset-a", {
```

Comment:
I think it's good to have this in the integration test, but it would be nice if the test actually verified the result somehow.

Reply:
Do you mean verifying what got uploaded includes the asset?

Reply:
Yes

Reply:
Perhaps you could use one of the Terraform functions to check for the [presence](https://www.terraform.io/docs/language/functions/fileexists.html)?